server/index.js

```diff
@@ -1,10 +1,35 @@
-import Fastify from 'fastify'
```

Comment:
i'll add a formatter and linter so that does not happen again, but can you revert the imports that did not actually change?